From ce526c0bff2ba1b7433f88e4e4310246d07617b2 Mon Sep 17 00:00:00 2001
From: Raimondas Galvelis <r.galvelis@acellera.com>
Date: Thu, 12 Jan 2023 12:54:49 +0100
Subject: [PATCH] Switch to setuptools

---
 python/setup.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/python/setup.py b/python/setup.py
index 024a67c..bd8e029 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -1,8 +1,6 @@
-from distutils.core import setup
-from distutils.extension import Extension
 import os
-import sys
 import platform
+from setuptools import Extension, setup
 
 openmm_dir = '@OPENMM_DIR@'
 atmmetaforce_header_dir = '@ATMMETAFORCEPLUGIN_HEADER_DIR@'
-- 
2.39.0

