From 5417545881470984fc3555b20efb8a7e1a2370b7 Mon Sep 17 00:00:00 2001
From: Raimondas Galvelis <r.galvelis@acellera.com>
Date: Thu, 12 Jan 2023 16:09:26 +0100
Subject: [PATCH] Fix setup.py install

---
 python/CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 173b2a4..22df3ad 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -21,9 +21,7 @@ set(ATMMETAFORCEPLUGIN_HEADER_DIR "${CMAKE_SOURCE_DIR}/openmmapi/include")
 set(ATMMETAFORCEPLUGIN_LIBRARY_DIR "${CMAKE_BINARY_DIR}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/setup.py ${CMAKE_CURRENT_BINARY_DIR}/setup.py)
 add_custom_command(TARGET PythonInstall
-    COMMAND "${PYTHON_EXECUTABLE}" setup.py build
-    COMMAND "${PYTHON_EXECUTABLE}" setup.py install --prefix=${CMAKE_INSTALL_PREFIX}
-    #COMMAND "${PYTHON_EXECUTABLE}" setup.py install
+    COMMAND "${PYTHON_EXECUTABLE}" setup.py install
     WORKING_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
 )
 
-- 
2.39.0

